src/main/java/co/cask/plugin/format/MultiTableConf.java

```diff
@@ -94,6 +94,12 @@
   @Description("List of tables NOT to fetch from the database. By default NONE of the tables will be black listed")
   private String blackList;
 
+  @Macro
+  @Nullable
+  @Description("SimpleDate format for converting the date fields in DB. " +
```

Comment:
date fields in DB. If specified, timestamp columns will be formatted into strings. If not specified, timestamp columns will be converted to long values.

Reply:
Fixed.